config/crds/migration.openshift.io_migplans.yaml

```diff
@@ -91,13 +95,18 @@ spec:
                   type: string
               type: object
             hooks:
+              description: Holds a reference to a MigHook along with the desired phase
+                to run it in.
               items:
                 description: MigPlanHook hold a reference to a MigHook along with
                   the desired phase to run it in
                 properties:
                   executionNamespace:
+                    description: Holds the name of the namespace where hooks should
+                      be implemented.
                     type: string
                   phase:
+                    description: Indicates the phase when the hooks will be executed.
```

Comment:
It might be useful to supply the list of available phases here.

```
	PreBackupHookPhase   = "PreBackup"
	PostBackupHookPhase  = "PostBackup"
	PreRestoreHookPhase  = "PreRestore"
	PostRestoreHookPhase = "PostRestore"
```